lib/rubocop/cop/style/file_name.rb

```diff
@@ -17,6 +17,14 @@ class FileName < Cop
 
         SNAKE_CASE = /^[\da-z_.?!]+$/
 
+        ACRONYM_NAME_MAPPINGS = {
+          'Cli'  => 'CLI',
+          'Dsl'  => 'DSL',
+          'Http' => 'HTTP',
+          'Rfc'  => 'RFC',
+          'Xml'  => 'XML'
+        }.freeze
```

Comment:
@pocke thanks for teach me helpful examples!

Add default allowed acronyms https://github.com/bbatsov/rubocop/pull/3798/commits/1efb5bb69ba8e6c3dd1a6790c50b2baa8793ffa0